compose_project_name

```diff
@@ -0,0 +1 @@
+ansibleplugintestingconjurhostidentity
```

Comment:
Just curious why this file was added. Is this required to run the dev environment?

Reply:
This is auto-generated file , Now start.sh script will remove it .